errors/index.js

```diff
@@ -0,0 +1,24 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err); // If it's not a custom error, pass it down the middleware chain
+  }
+};
+// PostgreSQL Errors
+exports.handlePsqlErrors = (err, req, res, next) => {
+  console.log("PostgreSQL Error Code:", err.code);
+  const psqlBadRequestCodes = ["22P02", "23502", "42703"];
```

Comment:
I think you may be including this console log intentionally to log out the PSQL error codes as they occur? 
This is not something I would recommend doing, as you build up your test suite the number of tests that invoke this console log will increase.
I would recommend keeping only the `500` block console log, because that is useful when developing. All other error handlers should be effectively "handling" the errors at which point there is no longer any need to log anything there. 

I can see you are also handling for `23502` and `42703`, is it possible to get those errors with the API as it currently is ?